<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config</class>
 <widget class="QWidget" name="config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>195</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>195</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>195</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/cancel.png</normaloff>:/resources/images/cancel.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="game_name_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="game_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/apply.png</normaloff>:/resources/images/apply.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="api_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>API</string>
   </property>
   <widget class="QRadioButton" name="dx9_radioButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DirectX 9</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dx_radioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DirectX (10,11,12)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="opengl_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenGL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
